src/components/AppBar/AppBar.jsx

```diff
@@ -83,13 +100,17 @@ const AppBar = ({
   };
 
   const TruncateUserName = (userName) => {
-    if (userName.length > 10) {
-      return `${userName.slice(0, 10)}....`;
+    try {
+      if (userName.length > 10) {
+        return `${userName.slice(0, 10)}....`;
+      }
+    } catch (e) {
```

Comment:
What is the error case here ? if it is `userName` not present let's do 
```
const TruncateUserName = (userName = 'Account')
```
rather than throwing and catching.

Reply:
The error case was if userName not present. I went forward with the above changes.